templates/newsroom/newsroom.js

```diff
@@ -0,0 +1,393 @@
+import {
+  div, section, input, span, a,
+} from '../../scripts/dom-helpers.js';
+
+// Function to fetch placeholders from JSON endpoint
+async function fetchPlaceholders() {
+  try {
+    const response = await fetch('https://main--fondationsaudemarspiguet--aemdemos.aem.page/placeholders.json');
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${response.status}`);
+    }
+    const data = await response.json();
+    return data; // Return the data object or empty object if not found
+  } catch (error) {
+    return {}; // Return empty object on error
+  }
+}
+
+// Function to detect current language
+function getCurrentLanguage() {
```

Comment:
an explicit function to detect the language isn't needed. Function `getLanguage()` already exists in `scripts.js`, suggest to import it and use directly